ocp.py

```diff
@@ -96,16 +96,7 @@ def __init__(self, train, track, optsDict={}):
         rho = train.rho
         totalMass = train.mass*rho
 
-        powerLosses = train.powerLosses
-
-        if opts.integrateLosses:
```

Comment:
You don't distinguish anymore between these two types of powerLosses based on this bool?